api/v1/logcollector_types.go

```diff
@@ -45,6 +45,13 @@ const (
 	CollectProcessPathDisable CollectProcessPathOption = "Disabled"
 )
 
+type EncryptionOption string
+
+const (
+	EncryptionNone EncryptionOption = "None"
```

Comment:
I think we want to set EncryptionNone in the controller if the user omits a value for Encryption. We do this for most other unspecified enums. Usually the method is called fillDefaults(...)

Reply:
Done